.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,59 @@
+name: Tag and Release on Merge from Release Branch
+
+on:
+  pull_request:
+    types: [closed]
```

Comment:
it works together with the if statement. from my pov release branches should always be merged.

Reply:
the if statement does not filter for merged/unmerged; and we cannot suppose that all bumping PRs will be merged. There may be the case that we created it, and then we close it because maybe it is still not time, or whatever other reason, a mistake, etc

Reply:
sorry, my bad, I read the IF statement wrong, you filter for it! my bad

Reply:
but so, just to confirm, ONLY the PRs that are merged will be treated with this workflow, right?

Reply:
yes exactly the workflow will run on closed prs / if merged and release in its name.